docs/development.md

```diff
@@ -44,8 +44,6 @@ Point your browser to <http://localhost:3000>
 
 Build the image and initialize the database:
 
-    lein uberjar
-    docker-compose build
```

Comment:
I think the _development_ instructions should show how to run your current version of rems. Should we have a different docker-compose file for demoing REMS using the public cscfi/rems image?

Reply:
Alternatively, I guess we could just take this part out of docs/development.md and move it to some other place like docs/installing-and-upgrading.md. Nobody's using docker-compose.yml for development currently as far as I know.

Reply:
Added second docker-compose file and moved the docker instructions to install doc